projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -183,25 +179,34 @@ const ProductCard = props => {
 		recordEvent( 'jetpack_myjetpack_product_card_fixconnection_click', {
 			product: slug,
 		} );
-		onFixConnection();
-	}, [ slug, onFixConnection, recordEvent ] );
+		navigateToConnectionPage();
+	}, [ slug, navigateToConnectionPage, recordEvent ] );
+
+	const ActionButtonInstance = actionButtonProps => (
+		<ActionButton
+			status={ status }
+			isFetching={ isFetching }
+			name={ name }
+			admin={ admin }
+			onActivate={ activateHandler }
+			onFixConnection={ fixConnectionHandler }
+			onManage={ manageHandler }
+			{ ...actionButtonProps }
```

Comment:
This is an anti-patterns. It hides things... We're refactoring a whole bunch of code so to not have to copy/paste props in the future. It's better to be clear and delcarative than to hide the properties to the readers of the code


* https://vhudyma-blog.eu/react-antipatterns-props-spreading/
* https://codeburst.io/react-anti-pattern-jsx-spread-attributes-59d1dd53677f